### ISSUE TITLE
changed image size

### ISSUE THREAD

**Issue by polcaruca** [ultralytics/ultralytics#21707]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hello!

Thanks for your great work on YOLO — I’ve been using it a lot and really appreciate it.

I ran into something I’m curious about.
I trained a YOLOv11n model on the SOD4B dataset without pretrained weights. 
And I set the image size to 1024.

What surprised me is that in the final YOLOv11n summary, the GFLOPs is still shown as 6.3, the same as with the default resolution. I thought increasing resolution should naturally increase GFLOPs for example, in EfficientNet, GFLOPs always go up with higher input sizes.

The training logs do show that images are being resized to 1024, so I’m wondering

Thanks for your time and help.

<img width="1166" height="953" alt="Image" src="https://github.com/user-attachments/assets/40d14510-3fc1-4e79-ac5e-b0db0d20e19f" />

**Comment by UltralyticsAssistant** [comment]
👋 Hello @polcaruca, thank you for your interest in Ultralytics 🚀! This is an automated response to help you get moving quickly—an Ultralytics engineer will follow up soon to assist further.

We recommend a visit to the Docs for comprehensive guidance where you can find many Python and CLI usage examples and where many of the most common questions may already be answered:
- Docs: https://docs.ultralytics.com/
- Python: https://docs.ultralytics.com/usage/python/
- CLI: https://docs.ultralytics.com/usage/cli/

If this is a 🐛 Bug Report, please provide a minimum reproducible example to help us debug it:
- MRE: https://docs.ultralytics.com/help/minimum-reproducible-example/

If this is a custom training ❓ Question, please provide as much information as possible, including:
- Exact command(s) used
- Model and version (e.g., YOLO11n)
- Ultralytics, PyTorch, CUDA versions
- Dataset details (images, object sizes), sample images, and training logs
Also verify you are following our Tips for Best Training Results:
- Training Tips: https://docs.ultralytics.com/guides/model-training-tips/

Join the Ultralytics community where it suits you best. For real-time chat, head to Discord 🎧. Prefer in-depth discussions? Check out Discourse. Or dive into threads on our Subreddit to share knowledge with the community:
- Discord: https://discord.com/invite/ultralytics
- Discourse: https://community.ultralytics.com/
- Subreddit: https://www.reddit.com/r/Ultralytics/

## Upgrade

Please upgrade to the latest ultralytics package including all requirements in a Python>=3.8 environment with PyTorch>=1.8 to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

- Requirements: https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml
- Python>=3.8: https://www.python.org/
- PyTorch>=1.8: https://pytorch.org/get-started/locally/

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including CUDA/CUDNN, Python and PyTorch preinstalled):

- Notebooks with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- Google Cloud Deep Learning VM. See GCP Quickstart Guide: https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/
- Amazon Deep Learning AMI. See AWS Quickstart Guide: https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/
- Docker Image. See Docker Quickstart Guide: https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/ <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all Ultralytics CI tests are currently passing. CI tests verify correct operation of all YOLO Modes and Tasks on macOS, Windows, and Ubuntu every 24 hours and on every commit.

To help us assist you faster regarding the GFLOPs display and image size:
- Please share your exact training command (including imgsz=1024), model selection (YOLO11n), and whether you used pretrained=False
- Provide the full console/model summary output and a short snippet of the training log where resizing to 1024 is shown
- Include your environment details (ultralytics version, PyTorch, CUDA, OS) and a small MRE if possible

Thanks again for the detailed context and screenshot 🙌

**Comment by glenn-jocher** [comment]
Thanks! The GFLOPs in the YOLO11n summary is computed once at model init using the default imgsz=640, so it won’t reflect your training imgsz=1024; FLOPs scale roughly with image area, so to see the correct value for 1024 use the model_info (set imgsz=1024) or get_flops functions described in the torch_utils reference, which will be about 2.56× higher than the 640 result (~16 GFLOPs). See model_info and get_flops in Reference for ultralytics/utils/torch_utils.py.

**Comment by Y-T-G** [comment]
You can use `model.model.info(imgsz=1024)` to get FLOPs at imgsz=1024
